gits/test/capabilities_test.py

```diff
@@ -492,16 +492,17 @@ def test_cap_kcub1(self):
         """
         term = self._terminal
 
+        self._check_cap_kcub1((0, 0))
+
         # The x position of the cursor is at the left-most position.
```

Comment:
The comment is incorrect, but it's not worth correcting it. Let's remove it at all.